### PR TITLE
CloudCore HA can be configured to use modules such as DeviceController

### LINKED ISSUE #2820: In current Cloud HA deployment, part of configuration Missing
**What happened**:
In current Cloud HA deployment（02-ha-configmap.yaml.example）, part of configuration Missing

**What you expected to happen**:
Can configure modules such as DeviceController DynamicController EdgeController Router SyncController

**How to reproduce it (as minimally and precisely as possible)**:
Follow this document(https://github.com/kubeedge/kubeedge/blob/master/build/cloud/ha/README.md) to deploy cloudcore HA

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- KubeEdge version(e.g. `cloudcore --version` and `edgecore --version`):
- <details><summary>Cloud nodes Environment:</summary>

  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>
- <details><summary>Edge nodes Environment:</summary>

  - edgecore version (e.g. `edgecore --version`):
  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>

### PR DESCRIPTION
**Pull request by Jw-Jm**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:
Provide an example of using modules such as DeviceController DynamicController in CloudCore HA deployment
**Which issue(s) this PR fixes**:

Fixes #2820

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: build/cloud/ha/02-ha-configmap.yaml.example [hunk 1]
```diff
@@ -46,3 +46,64 @@ data:
...
+          deviceModelEvent: 1
+          updateDeviceStatus: 1024
+        context:
+          receiveModule: devicecontroller
+          responseModule: cloudhub
+          sendModule: cloudhub
+        enable: true
+        load:
+          updateDeviceStatusWorkers: 1
+      dynamicController: {}
```
Comment (GsssC) [inline build/cloud/ha/02-ha-configmap.yaml.example:61]:
```suggestion
      dynamicController: {}
```
```suggestion
      dynamicController: 
        enable: false      
```

**Review by GsssC — COMMENTED** [review]
plz merge your commit into one : )
Thx for your contribution!

**Comment by Jw-Jm** [comment]
> plz merge your commit into one : )
> Thx for your contribution!

Thank you for review. opinions have been modified

> plz merge your commit into one : )
> Thx for your contribution!

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**